README.md

```diff
@@ -86,3 +86,12 @@ if ('serviceWorker' in navigator) {
     })
 }
 ```
+### Installation and testing
+
+* Clone the repo from here
+* cd to ipfs-service-worker
+* npm install
+* cd examples
+* npm run build
```

Comment:
It is necessary a `npm install` previously, in order to install `webpack`

Reply:
I don't understand this, I am running npm install prior to the build, in the ipfs-service-worker. 

To be honest, I think this example is much too complicated to be useful,  its completely unclear what "use-from-another-page" means and in my own implementation I've simplified it significantly to a single package.json, and a single webpack.config.js at the same level which builds the bundles in the dist directory.     I can't post that as a PR since its launching our own code (which includes IPFS) as the SW, rather than just IPFS.

Reply:
I agree with you that the example is pretty unclear. 

I believe it may be considered as an independent project, with its own dependencies. In this PR, there are still 2 [package.json](https://github.com/ipfs/ipfs-service-worker/pull/10/files#diff-4c2cd13b715374ef9285cf62654677f0R18). This way, with the implementation in this PR, it needs the `npm install`.